source/manual/alerts/high-memory-for-application.html.md

```diff
@@ -4,7 +4,7 @@ title: High memory for application
 parent: "/manual.html"
 layout: manual_layout
 section: Icinga alerts
-last_reviewed_on: 2017-11-22
+last_reviewed_on: 2018-11-02
```

Comment:
Is this supposed to be today's date?

Reply:
Apparently I can't tell the difference between august and november.  Updated.